knowledge-base/actions/irongut/editrelease/action-security.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Edit Release'
+# irongut/EditRelease
+github-token:
+  action-input:
+    input: token
+    is-default: true
```

Comment:
The `is-default` should be false since it is not using a default value here: https://github.com/irongut/EditRelease/blob/7601329922c975d88d0943603a881777341683fb/action.yml#L10